diff -p -up mysql-connector-net-6.2.4/MySql.Web/Providers/MySql.Web.csproj.orig mysql-connector-net-6.2.4/MySql.Web/Providers/MySql.Web.csproj
--- mysql-connector-net-6.2.4/MySql.Web/Providers/MySql.Web.csproj.orig	2010-08-18 14:29:36.000000000 +0200
+++ mysql-connector-net-6.2.4/MySql.Web/Providers/MySql.Web.csproj	2010-10-10 11:49:51.000000000 +0200
@@ -56,7 +56,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <NoWarn>1699</NoWarn>
-    <DocumentationFile>bin\release\webdocs.xml</DocumentationFile>
+    <DocumentationFile>bin\Release\webdocs.xml</DocumentationFile>
     <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
